enterprise_access/apps/bffs/context.py

```diff
@@ -197,6 +207,17 @@ def _initialize_common_context_data(self):
         if not self.enterprise_customer_uuid:
             self._enterprise_customer_uuid = self.enterprise_customer.get('uuid')
 
+        # Initialize the secured algolia api keys metadata derived from enterprise catalog
+        try:
+            self._initialize_secured_algolia_api_keys()
+        except Exception:  # pylint: disable=broad-except
+            logger.exception(
```

Comment:
Two questions/considerations:
1. `self. _initialize_secured_algolia_api_keys()` itself wraps its entire function body in a try/except, where I don't believe an error would not be surfaced to this try/except given it was already caught within `self. _initialize_secured_algolia_api_keys()`. Given this, would it make sense to keep only 1 layer of try/except for this method?
2. When handling an error related to `self. _initialize_secured_algolia_api_keys()`, do we want the error to be returned in the BFF API response in its `errors` list?

Reply:
[clarification] For (2), if we want the error to be surfaced in the BFF API response itself, `self.add_error(...)`.